pattern-2/bosh-release/jobs/keymanager/templates/ctl.erb

```diff
@@ -87,6 +94,14 @@ case $1 in
     # Create symbolic link for logging
     ln -s $WSO2_APIM_HOME/repository/logs/ ${log_dir}
 
+    #write the cert details to a file
+    echo ${CA_CERT} >> tmp.cert
+
+    sudo ${KEYTOOL} -import -noprompt -trustcacerts -alias ${ALIAS} -file tmp.cert \
```

Comment:
Do we need root permission for this?

Reply:
Only the root user has the write permission to the files inside the /var/vcap/packages/ directory. But as this script run by the root, this is not necessary. Therefore removed with 784e083ddf9371ed582c141c4904743a0d6ddb7b